f8a_worker/storages/s3.py

```diff
@@ -27,11 +27,12 @@ def __init__(self, aws_access_key_id=None, aws_secret_access_key=None, bucket_na
         super().__init__()
         self._s3 = None
 
-        self.region_name = os.getenv('AWS_S3_REGION', region_name) or self._DEFAULT_REGION_NAME
+        self.region_name = configuration.AWS_S3_REGION or region_name or self._DEFAULT_REGION_NAME
```

Comment:
Could we move the `DEFAULT_REGION_NAME` (and perhaps also the `DEFAULT_ENCRYPTION`) to `F8AConfiguration` as well ?

Reply:
I was thinking about it, but I was not sure. I will move it.

Reply:
I think I've changed my mind after all :-)
You're perhaps right, that it can stay in the `AmazonS3` class since it's not used anywhere else.
So feel free to revert the last commit and I'll merge it.